presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -84,6 +94,48 @@ public FilterNode filter(Expression predicate, PlanNode source)
         return new FilterNode(idAllocator.getNextId(), source, predicate);
     }
 
+    public TableScanNode tableScan(List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments)
+    {
+        Expression originalConstraint = null;
+        return new TableScanNode(
+                idAllocator.getNextId(),
+                new TableHandle(
+                        new ConnectorId("testConnector"),
+                        new TestingTableHandle()),
+                symbols,
+                assignments,
+                Optional.empty(),
+                TupleDomain.all(),
+                originalConstraint
+        );
+    }
+
+    public TableFinishNode tableDelete(SchemaTableName schemaTableName, PlanNode deleteSource, Symbol deleteRowId)
```

Comment:
I need to match finish node's output symbols with output symbols of produced values, in `TestRemoveEmptyDelete.test`.